src/main/java/mate/academy/spring/model/dto/CinemaHallRequestDto.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.model.dto;
+
+import java.io.Serializable;
+
+public class CinemaHallRequestDto implements Serializable {
```

Comment:
```suggestion
public class CinemaHallRequestDto {
```
the same for other dtos